### DISCUSSION TITLE
Root endpoint

### DISCUSSION THREAD

**Discussion by massimoliani** [directus/directus#9130] [Archive]
Is there a way to modify the standard redirection of root path / to /admin to an endpoint of directus that serve a vue app?

The idea is to have a simplified interface for final user front end and leave direct us admin app and backend for administrator of the organization.

Thank you.

**Comment by rijkvanzanten** [comment]
See ROOT_REDIRECT ~~https://docs.directus.io/reference/environment-variables/#general~~ 👍🏻 

EDIT: Moved to https://docs.directus.io/configuration/config-options/#general

**Reply by massimoliani** [reply]
Thanks
